ale_linters/haml/hamllint.vim

```diff
@@ -1,5 +1,17 @@
-" Author: Patrick Lewis - https://github.com/patricklewis
+" Author: Patrick Lewis - https://github.com/patricklewis, thenoseman - https://github.com/thenoseman
 " Description: haml-lint for Haml files
+"
+" Options:
+" g:ale_haml_hamllint_options can be set and will be passed as options to
+" haml-lint
+" eg. let g:ale_haml_hamllint_options = '-c ~/use-my.haml-lint.yml'
+
+let g:ale_haml_hamllint_options =
+\   get(g:, 'ale_haml_hamllint_options', '')
+
+function! ale_linters#haml#hamllint#GetCommand(buffer) abort
+    return 'haml-lint ' . g:ale_haml_hamllint_options . ' %t'
```

Comment:
I recommend only adding the extra space on the left if the options aren't empty. That's better for tests. Have a look at some existing linters for examples.